isites_migration/migrations/0001_initial_lti_permissions.py

```diff
@@ -28,17 +29,26 @@
 
 
 def create_lti_permissions(apps, schema_editor):
-    LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    try:
+        LtiPermission = apps.get_model('lti_permissions', 'LtiPermission')
+    except LookupError:
+        # If the LtiPermission table doesn't exist, return from here
+        return
+
     for permission in LTI_PERMISSIONS:
         lti_permission = LtiPermission(**permission)
         lti_permission.save()
 
 
 class Migration(migrations.Migration):
 
-    dependencies = [
-        ('lti_permissions', '0001_initial')
-    ]
+    dependencies = []
+
+    # tlt-2650: we need to check whether lti_permissions is part of
+    # INSTALLED_APPS and skip this migration if the LtiPermission model is
```

Comment:
We might want to clarify this part of the code -- we're not skipping the migration, we're just allowing it to proceed if `lti_permissions` is no longer in the list of `INSTALLED_APPS`. I guess this would be more accurate if we simply set `operations = []` below.